firefox_puppeteer/ui/toolbars.py

```diff
@@ -55,26 +65,27 @@ def menu_button(self):
         return self.marionette.find_element(By.ID, 'PanelUI-menu-button')
 
 
-class LocationBar(BaseLib):
+class LocationBar(UIBaseLib):
     """Provides access to and methods for the DOM elements contained in the
     locationbar (the text area of the ui that typically displays the current url)."""
 
-    dtds = ["chrome://branding/locale/brand.dtd",
-            "chrome://browser/locale/browser.dtd"]
-
-    def __init__(self, *args, **kwargs):
-        BaseLib.__init__(self, *args, **kwargs)
-        # TODO: A "utility" module that sets up the client directly would be
-        # useful here.
-        self.l10n = L10n(self.get_marionette)
-        self.keys = Keys(self.get_marionette)
+    def __init__(self, marionette_getter, window):
+        UIBaseLib.__init__(self, marionette_getter, window)
+        
+        self._autocomplete_results = None
+        self._identity_popup = None
 
-    @use_class_as_property('ui.toolbars.AutocompleteResults')
+    @property
     def autocomplete_results(self):
         """Provides access to and methods for the location bar
         autocomplete results.
 
         See the :class:`AutocompleteResults` reference."""
+        if not self._autocomplete_results:
+        	from .toolbars import AutocompleteResults
+        	self._autocomplete_results = AutocompleteResults(lambda: self.marionette, self.window)
```

Comment:
You have still tabulators enabled for indentation. Please disable that in your editor and use 4 chars instead.